cool-core/src/main/java/com/nus/cool/core/cohort/ExtendedCohortQuery.java

```diff
@@ -80,13 +80,12 @@ public void setFillWithNextObserved(boolean fillWithNextObserved) {
     private String userId;
 
     @JsonIgnore
-    public boolean isValid() throws IOException {
-        if ((birthSequence != null) &&
-            (birthSequence.isValid()) &&
-            (dataSource != null) &&
-            (ageField != null) &&
-            (measure != null)) return true;
-        else throw new IOException("[x] Invalid cohort query.");
+    public boolean isValid() {
```

Comment:
Please revert this change, i.e., keep the original isValid() function with IOException.